docs/trulens_eval/evaluation/feedback_evaluations/comprehensiveness_benchmark.ipynb

```diff
@@ -0,0 +1 @@
+../../../../trulens_eval/trulens_eval/tests/comprehensiveness_benchmark.ipnb
```

Comment:
think symlink isn't quite right? When I build docs on this branch, I get:

FileNotFoundError: [Errno 2] No such file or directory: '/Users/jreini/Desktop/development/trulens/docs/trulens_eval/evaluation/feedback_evaluations/comprehensiveness_benchmark.ipynb'

Reply:
fixed - my bad there's a weird path typo and now the symlink should work properly